src/components/VersionFileViewer/index.spec.tsx

```diff
@@ -41,61 +42,48 @@ import VersionFileViewer, { ItemTitles, PublicProps } from '.';
 
 describe(__filename, () => {
   const getInternalVersionAndFile = ({
-    store = configureStore(),
-    path = 'background.js',
+    contentOrDiff = 'content',
```

Comment:
Would it make sense to use constants (or an enum) here? This seems to only be used in the test file, though.

Reply:
Yeah, this is just a helper function for the tests, so I'm not sure it's worth doing that. WDYT?

Reply:
it's fine, I guess

Reply:
You sounded so sad about this, I decided to change it anyway. It wasn't much effort.